src/lambda/release.js

```diff
@@ -0,0 +1,77 @@
+import fetch from 'node-fetch';
+
+exports.handler = async event => {
+	const { version, url } = await fetchRelease(
+		event.queryStringParameters.repo || 'preactjs/preact'
```

Comment:
Nit: can we guard against folks hitting this with arbitrary repo names? Could just allow only `preactjs/` org repos:

```suggestion
		`preactjs/${event.queryStringParameters.repo || 'preact'}`
```